From 6a0e16e85d391e166684750a325dc3568deee997 Mon Sep 17 00:00:00 2001
From: Zhangbin Tong <zebulun.tong@rock-chips.com>
Date: Tue, 29 May 2018 15:03:19 +0800
Subject: [PATCH] ARM: dts: rk3128h-box: Limit vdd_arm minimum voltage to 1.2v
 during early kernel boot

vdd_arm can supply multiple consumers(arm/dmc/gpu/rkvdec), we need to keep
a voltage meets all consumers before all consumers complete initialization.

Change-Id: Ie5f3c1e879931fe69f8b807385c8e97f5cdf5d93
Signed-off-by: Zhangbin Tong <zebulun.tong@rock-chips.com>
---
 arch/arm/boot/dts/rk3128h-box.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm/boot/dts/rk3128h-box.dts b/arch/arm/boot/dts/rk3128h-box.dts
index 1facaf6f3acc..f69a4744b318 100644
--- a/arch/arm/boot/dts/rk3128h-box.dts
+++ b/arch/arm/boot/dts/rk3128h-box.dts
@@ -213,7 +213,7 @@
 	vdd_arm: vdd-arm-regulator {
 		compatible = "pwm-regulator";
 		rockchip,pwm_id = <1>;
-		rockchip,pwm_voltage = <1200000>;
+		regulator-init-microvolt = <1200000>;
 		pwms = <&pwm1 0 5000 1>;
 		regulator-name = "vdd_arm";
 		regulator-min-microvolt = <950000>;
-- 
2.35.3

